spec/controllers/posts_controller_spec.rb

```diff
@@ -19,5 +20,10 @@
       get :index
       expect(response).to have_http_status(200)
     end
+
+    scenario "renders posts template" do
```

Comment:
Use `it` in unit tests.

Reply:
(The strategy here is noticing what it does above, and staying consistent)

Reply:
now uses 'it'